ui/src/app/components/object-detail/object-detail.component.scss

```diff
@@ -186,8 +186,8 @@
     flex-direction: row;
     justify-content: left;
     align-items: center;
-    .error {
-      color: #eb0000;
+    .note{
```

Comment:
the icons are being shown as blue and its still showing error icon: https://drive.google.com/file/d/1fQM61eP_t43wnJd0nQMk7vurWp8gt88z/view?usp=sharing&resourcekey=0-M5NvbEfU-pgECOqYE56i7w

Reply:
Initially we made it according to the figma design, but after the bug reported we made these changes, can you please suggest what icon can we show over there.

Reply:
the bug meant using same icon not colour, in my opinion we can use warning icon in both the places, let me know if you have any better suggestion.

Reply:
Updated